### PR TITLE
Add install analysis for 4.17 and 4.18 multi payload

### PR DESCRIPTION
**Pull request by KeerthanaAP**

### CODE REVIEW

**Comment by KeerthanaAP** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.17-install-analysis-all-multi-p-p

File: core-services/release-controller/_releases/priv/release-ocp-4.17-multi.json [hunk 1]
```diff
@@ -212,6 +212,13 @@
             "prowJob": {
                 "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-heavy-build-ovn-remote-libvirt-multi-p-p-priv"
             }
+        },
+        "install-analysis-multi-p-p": {
```
Comment (tvardema) [inline core-services/release-controller/_releases/priv/release-ocp-4.17-multi.json:216]: I don't think we're supposed to update the `/priv` directory files... Were these generated automatically?

**Pull request merged**